### PR TITLE
2473 document a page for introducing premium edition and complete list of premium features

### LINKED ISSUE #2473: Document a page for introducing Premium edition and complete list of premium features
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

Document a page for introducing Premium edition and complete list of premium features

### Reference

_No response_

### LINKED ISSUE #2564: Change required: label time travel as a premium feature
https://docs.risingwave.com/docs/next/time-travel-queries/

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Description

Introduction for Premium Edition and offer complete list of premium features

## Related doc issue

resolves https://github.com/risingwavelabs/risingwave-docs/issues/2564
 
resolves https://github.com/risingwavelabs/risingwave-docs/issues/2473

## Rendered preview
https://pr-2492.d2fbku9n2b6wde.amplifyapp.com/docs/next/rw-premium-edition-intro/

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

**Comment by ShanlanLi** [comment]
@neverchanje This is the premium version introduction page of the doc([preview](https://pr-2492.d2fbku9n2b6wde.amplifyapp.com/docs/next/rw-premium-edition-intro/)), which is based on the INTRODUCING RISINGWAVE PREMIUM PDF. Regarding the list of premium features, I wonder:

1. Does the feature list align with yours?
2. In terms of the presentation of the premium feature, there are two ways, which one may be better?

2.1 List out all the planned features, and add links to the existing ones, like:
image

2.2 Just list the existing features

**Comment by lmatz** [comment]
>Does the feature list align with yours?

https://github.com/risingwavelabs/risingwave/blob/main/src/license/src/feature.rs#L45-L60

The features mentioned here are all premium features


> In terms of the presentation of the premium feature, there are two ways, which one may be better?

Let's go with "Just list the existing features". Not all planned features will be implemented, sometimes priority re-adjusted based on customer feedback. I will remove the "planned features" from the Notion going forward.



BTW, @st1page and I will take over the docs about the kernel, please tag us in the future.

**Comment by lmatz** [comment]
#2564

**Comment by ShanlanLi** [comment]
> > Does the feature list align with yours?
> 
> https://github.com/risingwavelabs/risingwave/blob/main/src/license/src/feature.rs#L45-L60
> 
> The features mentioned here are all premium features
> 
> > In terms of the presentation of the premium feature, there are two ways, which one may be better?
> 
> Let's go with "Just list the existing features". Not all planned features will be implemented, sometimes priority re-adjusted based on customer feedback. I will remove the "planned features" from the Notion going forward.
> 
> BTW, @st1page and I will take over the docs about the kernel, please tag us in the future.

Thanks, will just retain the existing one

**Comment by WanYixian** [comment]
BTW，premium edition feature means enterprise features / paid features right? If so, I'll add [LicenseManager and feature tiers](https://github.com/risingwavelabs/risingwave/pull/17396) to this doc later.

File: docs/deploy/manage-secrets.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ slug: /manage-secrets
   <link rel="canonical" href="https://docs.risingwave.com/docs/current/manage-secrets/" />
 </head>
 
+
+:::tip Premium Edition Feature
+This feature is only available in the premium edition of RisingWave. The premium edition offers additional advanced features and capabilities beyond the free and community editions.  For a complete list of premium features and more information about the premium edition, please visit our [RisingWave Premium introduction](/rw-premium-edition-intro.md#complete-list-of-premium-features) page. If you have any questions about upgrading to the premium edition, please contact our sales team at [sales@risingwave-labs.com](mailto:sales@risingwave-labs.com).
```
Comment (hengm3467) [inline docs/deploy/manage-secrets.md]: For the description about the separate page, you do not need to call out a link is a page. Just say "please see [RisingWave Premium Edition](link-here.md)" is enough.

**Review by WanYixian — APPROVED** [review]

**Pull request merged**